Gemfile

```diff
@@ -61,6 +61,7 @@ group :development, :test do
   gem "ruby-debug-passenger"
   gem "selenium-webdriver"
   gem "jettywrapper"
+  gem "fcrepo_wrapper"
```

Comment:
Don't know anything about this gem so bare with me,  do these rake tasks ever need to be ran in production environment? Does the wrapper only provide rake tasks/cli? if so you could potentially set it to `required: false` as it doesn't needed to be loaded in every process?